ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -311,6 +311,20 @@ describe('a4a_config', () => {
             'element in ', EXTERNAL_BRANCHES[branch]).to.be.false;
       }
     });
+
+    it(`should force controlMeasureOnRender param from URL ` +
+       `when pattern=${urlBase}`, () => {
+      win.location.search = urlBase.replace('PARAM', 'a4a:3');
+      // Should not register as 'A4A enabled', but should still attach the
+      // controlMeasureOnRender experiment ID.
+      expect(googleAdsIsA4AEnabled(win, element, EXP_ID, EXTERNAL_BRANCHES,
+          INTERNAL_BRANCHES), 'googleAdsIsA4AEnabled').to.be.false;
```

Comment:
don't quite get the assertion here. What the second `'googleAdsIsA4AEnabled'` does?